api/src/main/java/org/pmiops/workbench/workspaces/WorkspaceServiceImpl.java

```diff
@@ -804,10 +802,12 @@ public void updateWorkspaceBillingAccount(DbWorkspace workspace, String newBilli
                         DbStorageEnums.workspaceActiveStatusFromStorage(
                                 row.getWorkspaceActiveStatus())
                             .toString())
-                    .addTag(
-                        MetricLabel.DATA_ACCESS_LEVEL,
-                        DbStorageEnums.dataAccessLevelFromStorage(row.getDataAccessLevel())
-                            .toString())
+                    // TODO replace with AccessTier RW-6137
+                    //                    .addTag(
+                    //                        MetricLabel.DATA_ACCESS_LEVEL,
+                    //
+                    // DbStorageEnums.dataAccessLevelFromStorage(row.getDataAccessLevel())
+                    //                            .toString())
```

Comment:
Does this affect the accuracy of this metric since the tag will be empty?

Reply:
@freemabd  Funny thing: this just occurred to me independently.  I pushed a temp fix: always set to Registered